js/atree_constants.js

```diff
@@ -9950,7 +9950,14 @@ const atrees = {
             "properties": {
                 "clone": 3
             },
-            "effects": []
+            "effects": [
+
+                    {
+                        "type": "raw_stat",
+                        "toggle": "Activate Clones",
+                        "bonuses": [{ "type": "stat", "name": "defMult.Clone", "value": 70}]
+                    }
```

Comment:
ingame value is 80% not 70% (but we do not know the proc chance); id leave it at 80 for now